Dockerfile

```diff
@@ -1,8 +1,19 @@
-FROM alpine
+FROM alpine:edge AS build
+WORKDIR /go/src/github.com/prebid/prebid-server/
+RUN apk add -U --no-cache go git fftw-dev musl-dev dep
```

Comment:
Not a big deal, but... why is `fftw-dev` in here?

We're not doing any fast fourier transforms... and the `docker build .` seems to work without it.